libs/server/Resp/BasicCommands.cs

```diff
@@ -24,6 +24,9 @@ bool NetworkGET<TGarnetApi>(byte* ptr, ref TGarnetApi storageApi)
             if (storeWrapper.serverOptions.EnableScatterGatherGet)
```

Comment:
should SG run if useAsync? It seems SG's SendAndReset can run while AsyncGetProcessor's is

Reply:
SG runs synchronously on the main incoming network packet thread, so it would preclude the running of the AsyncGetProcessor at the same time.